wherehows-web/app/templates/components/datasets/owners/suggested-owner-card.hbs

```diff
@@ -0,0 +1,29 @@
+<section class="dataset-authors-suggested__card__owner-info">
```

Comment:
suggestion: in the future, let's keep the BEM nesting to max depth of 3 where possible, there are some classes that go up to 6 here. BEM nesting does not necessarily have to mirror HTML structure, the approach should be more in terms of the component/Block and how each Element relates to that component.

Reply:
I'll see what I can do short term and keep that in mind.